graphql/schema.js

```diff
@@ -40,8 +41,13 @@ type Page {
   author: String
   content: String
   lead: String!
+  type: String!
   # DD.MM.YYYY HH:MM
-  created: String
```

Comment:
in order to avoid breaking things (e.g. browser extensions using the GraphQL-API) I would leave the field in. Since it's optional (without `!`) you can still remove the logic in the mapper and never fill it anymore. And add a deprecated comment with `#` above it

Reply:
Done